ui/__tests__/util/policy.test.js

```diff
@@ -0,0 +1,42 @@
+import Policy from '../../util/policy';
+
+function policy(props = {}) {
+  return new Policy({
+    id: 1,
+    has_docnode: false,
+    omb_policy_id: 'M-16-19',
+    ...props,
+  });
+}
+
+describe('Policy', () => {
+  it('is frozen', () => {
+    expect(Object.isFrozen(policy())).toBe(true);
```

Comment:
I might be going overboard with this freezing thing, let me know what you think!